### DISCUSSION TITLE
video stream is not showing on video player but audio is present

### DISCUSSION THREAD

**Discussion by ghost** [mifi/lossless-cut#1624] [Support]
Any ideas on how to fix? Thanks

-----------------

**some observations/info**

- The source of the video is a Bluray disk. I ripped the disc  using MakeMKV rip (.m2ts output file)
- The single .m2ts file contains 4 anime episodes.
- I encoded the video using Handbrake + H265 + AMC VCE. I used MP4 and MKV output format.
- I have no issues watching the source and Handbrake output videos using the video players listed below. I can forward or go back to previous frame (if available).
- The 1st episode that I cut has no problems playing using the video players listed below.
- Only the succeeding segments/episodes won't show video. But I can hear the audio.
- I loaded the output videos into mediainfo-gui and I can see all of them has a video stream.
- I tried with keyframe cut mode enabled and disabled. Same results.
- I used ffplay, mpv and VLC to watch the videos.
- When I used MP4 as output format in Handbrake, the segment that will not play on video players will play in Chrome.
- As another test, I encoded the same video using H265 (regular/not using AMD VCE) and lossless-cut was able to cut this properly. No issues watching the exported segment.
- As a final test, I reencoded (using AMD VCE x265) another Bluray disk and I get the same results. 1st episode is watchable but the succeeding ones are not.

**Comment by mifi** [comment]
do you mean when opening the source file in losslesscut, you get only audio? or after cutting, or after cut/merge when playing back the result? maybe you can provide some more info (see here: https://github.com/mifi/lossless-cut/issues/new?assignees=&labels=&projects=&template=file-issue.yml )
